4.5 release page (#1138)
godotengine/godot-website: collections/_release_4_5/entry-general-editor-toggleable-inspector-sections.md

---
type: entry
section: general
subsection: editor
rank: 4
importance: 2
anchor: toggleable-inspector-sections
title: Toggleable inspector sections
blockquote: Go Go Inspector Section!
text: |

>   User experience upgrade. We replaced the enabled property within groups with an intuitive checkbox beside the section name. You can now easily see which sections are enabled, even collapsed ones.

Reviewer: ```suggestion
  As a user experience upgrade, we replaced the enabled property within groups with an intuitive checkbox beside the section name. You can now easily see which sections are enabled, even collapsed ones.
```

>   User experience upgrade. We replaced the enabled property within groups with an intuitive checkbox beside the section name. You can now easily see which sections are enabled, even collapsed ones.

Reviewer 2: ```suggestion
  User experience upgrade. We replaced the "enabled" property within groups with an intuitive checkbox beside the section name. You can now easily see which sections are enabled, even collapsed ones.
```

Author: I used instead code quotes.

>   Plugin developers can implement this feature in their projectrs using the new [`PROPERTY_HINT_GROUP_ENABLE`](https://docs.godotengine.org/en/4.5/classes/class_@globalscope.html#class-globalscope-constant-property-hint-group-enable).

Reviewer: ```suggestion
  Plugin developers can implement this feature in their projects using the new [`PROPERTY_HINT_GROUP_ENABLE`](https://docs.godotengine.org/en/4.5/classes/class_@globalscope.html#class-globalscope-constant-property-hint-group-enable).
```

>   Plugin developers can implement this feature in their projectrs using the new [`PROPERTY_HINT_GROUP_ENABLE`](https://docs.godotengine.org/en/4.5/classes/class_@globalscope.html#class-globalscope-constant-property-hint-group-enable).

Reviewer 2: ```suggestion
  Plugin developers can use this feature in their projects using the new [`PROPERTY_HINT_GROUP_ENABLE`](https://docs.godotengine.org/en/4.5/classes/class_@globalscope.html#class-globalscope-constant-property-hint-group-enable).
```
The feature is already implemented
